dist/index.d.ts

```diff
@@ -67,7 +73,8 @@ type OutputType = keyof OutputByType;
   ): void;
 
   export function zip(
-    geojson: GeoJSON.FeatureCollection, 
-    options: DownloadOptions & ZipOptions = { compression: 'STORE', type: 'base64' }, 
```

Comment:
I liked the way this was written. It keeps the implementations cleaner and more legible in my opinion.

Reply:
I am also considering that zip options should not have to worry about DownloadOptions. I apologize for bombarding you with messages, I am just hoping to resolve to the functionality it had when I created my PR.